src/test/kotlin/no/nav/syfo/application/api/OppfolgingsplanSystemApiSpek.kt

```diff
@@ -54,6 +81,11 @@ class OppfolgingsplanSystemApiSpek : Spek({
                             }
                         ) {
                             response.status() shouldBeEqualTo HttpStatusCode.OK
+                            runBlocking {
+                                val result = dialogmeldingSendCronjob.dialogmeldingSendJob()
+                                result.failed shouldBeEqualTo 0
+                                result.updated shouldBeEqualTo 1
+                            }
```

Comment:
Er det riktig forstått at vi tidligere har sendt oppfølgingsplanen på mq som en del av post-kallet, men at den nå blir lagt til bestilling og legges på mq som en del av en cronjob?
Hadde det ikke da vært bedre om denne testen verifiserer at man ikke legger noe på mq? I stedet for å kjøre kode (cronjobben) som ikke har noe med endepunktet å gjøre? Det samme gjelder `personOppfolgingsplanApiSpek` 🤔

Reply:
👍 Her bør man kanskje heller teste at API-kallet oppretter en bestilling i databasen (og at det ikke legges noe på MQ som @JMLindseth sier)

Reply:
Ja, det er riktig at det nå blir cronjob'en som faktisk sender oppfølgingsplanene. 

Skal se om testene kan forbedres, og det er forsåvidt greit å teste på at bestillingen opprettes her. Men det er jo også et poeng å teste at koden som tar en bestilling for en oppfølgingsplan faktisk klarer å sende den til MQ-køen, så må nok ha en test for dette i tillegg. Men det hører kanskje bedre hjemme i cronjob-testen.